src/Features/CSharp/Portable/BraceCompletion/CurlyBraceCompletionService.cs

```diff
@@ -370,8 +370,13 @@ public override AbstractFormattingRule WithOptions(AnalyzerConfigOptions options
                 return new BraceCompletionFormattingRule(_indentStyle, cachedOptions);
             }
 
-            public override AdjustNewLinesOperation? GetAdjustNewLinesOperation(in SyntaxToken previousToken, in SyntaxToken currentToken, in NextGetAdjustNewLinesOperation nextOperation)
+            private static bool? NeedsNewLine(in SyntaxToken currentToken, CachedOptions options)
```

Comment:
```suggestion
            private static bool? NeedsNewLine(in SyntaxToken currentToken, in CachedOptions options)
```

Reply:
@sharwell In a recent refactoring, `CachedOptions` was completely removed and a new class (as opposed to struct) is introduced. So I think no action is needed for this comment now.